contrib/mypy/src/python/pants/contrib/mypy/tasks/mypy_task.py

```diff
@@ -121,7 +164,6 @@ def execute(self):
       with open(sources_list_path, 'w') as f:
         for source in sources:
           f.write('{}\n'.format(source))
-
       # Construct the mypy command line.
       cmd = ['--python-version={}'.format(interpreter_for_targets.identity.python)]
```

Comment:
Can change this to an f-string. We're usually doing this when reworking a file for general code cleanup.
```suggestion
      cmd = [f'--python-version={interpreter_for_targets.identity.python}']
```

Would be great if you could do this for all strings in the file by searching for `.format()`.